### ISSUE TITLE
Command escaping doesn't work on Windows when there's a space in the path

### ISSUE THREAD

**Issue by zaggino** [IndigoUnited/node-cross-spawn#9]
```
var spawn = require('cross-spawn'); // version 0.2.6

// var command = "node"; // works fine
var command = "c:\\Program Files\\iojs\\node.exe";
// fails even though path is valid
// error is Cannot find module 'D:\Dev\test\Files\iojs\node.exe'
// when executed from 'D:\Dev\test\' directory

var args = ["--version"];

var spawned = spawn(command, args);

spawned.stdout.on('data', function (data) {
  console.log('stdout: ' + data);
});

spawned.stderr.on('data', function (data) {
  console.log('stderr: ' + data);
});

spawned.on('close', function (code) {
  console.log('child process exited with code ' + code);
});

```

**Comment by zaggino** [comment]
Debugging, actual parameters send to the original child_process.spawn are:
```
command: "C:\Windows\system32\cmd.exe"
args: [ '/s',
  '/c',
  '"c:\\Program^ Files\\iojs\\node.exe "--version""' ]
```

**Comment by zaggino** [comment]
I'd put up a PR to fix this, but I'm not sure how... but I think `cmd.exe` shoulnd't be there in this case.

**Comment by satazor** [comment]
@zaggino so it works if you execute from c:\ but not from d:\?

**Comment by zaggino** [comment]
No, doesn't matter where you execute it from. I've moved the test files to `C:\test`:
```
Zaggi@ZAGGI-PC /c/test
$ pwd
/c/test

Zaggi@ZAGGI-PC /c/test
$ node test.js
C:\Windows\system32\cmd.exe
[ '/s',
  '/c',
  '"c:\\Program^ Files\\iojs\\node.exe "--version""' ]
stderr: module.js:318
    throw err;
          ^
Error: Cannot find module 'C:\test\Files\iojs\node.exe'
    at Function.Module._resolveFilename (module.js:316:15)
    at Function.Module._load (module.js:258:25)
    at Function.Module.runMain (module.js:451:10)
    at startup (node.js:123:18)
    at node.js:868:3

child process exited with code 1
```

**Comment by zaggino** [comment]
I don't know the exact magic behind using `cmd.exe` but it probably shouldn't be used for other `.exe` files?

**Comment by satazor** [comment]
Should be fixed now! Thanks for the report.

**Comment by zaggino** [comment]
Thanks, confirmed fixed in `0.2.8`
